airflow/providers/common/sql/operators/sql.py

```diff
@@ -258,6 +258,9 @@ def _process_output(self, results: list[Any], descriptions: list[Sequence[Sequen
             self.log.info("Operator output is: %s", results)
         return results
 
+    def _run_output_processing(self) -> bool:
```

Comment:
This name sounds like the function actually runs the processing, but in reality it only returns whether processing _should_ be done. Maybe prefix the name with `should`?

Reply:
Right. Indeed.